dynatrace-aws-logs.sh

```diff
@@ -199,13 +199,15 @@ arguments:
     UseExistingActiveGate="$USE_EXISTING_ACTIVE_GATE" TenantId="$TENANT_ID" DynatracePaasToken="$TARGET_PAAS_TOKEN" \
     --no-fail-on-empty-changeset
 
+  # SHOW OUTPUTS
+  aws cloudformation describe-stacks --stack-name "$STACK_NAME" --query "Stacks[0].Outputs"
+
   LAMBDA_ARN=$(aws cloudformation describe-stacks --stack-name "$STACK_NAME" \
      --query "Stacks[0].Outputs[?OutputKey=='LambdaArn'][OutputValue]" --output text)
 
-  aws lambda update-function-code --function-name "$LAMBDA_ARN" --zip-file fileb://"$LAMBDA_ZIP_NAME" > /dev/null
+  echo; echo "Deploying Lambda changes to $LAMBDA_ARN"
```

Comment:
I am not sure if this is needed, how is this output helpful? 
As a user I only care if it works. I mean I don't need this output values. The script will fail if the lambda upload fails, right? Why would I need this additional output as an end user?

Reply:
I discussed this on some daily already, and I think @wojtasskorcz has agreed, that it is misleading if someone wants to run and update, and for some reason, changes since last release are only in Lambda, not in the infrastructure. You then get only a message that no changes were deployed, like this:

```
[cloudshell-user@ip-10-1-157-79 ~]$ ./dynatrace-aws-logs.sh deploy --use-existing-active-gate true

Deployment script will use following parameters:
TARGET_URL="https://ec2-18-232-177-51.compute-1.amazonaws.com:9999/e/jxw01498", TARGET_API_TOKEN=, USE_EXISTING_ACTIVE_GATE="true", TARGET_PAAS_TOKEN=, REQUIRE_VALID_CERTIFICATE="false", STACK_NAME="BeluPerfLogsCLientSecond"
Deploying stack BeluPerfLogsCLientSecond

Waiting for changeset to be created..

No changes to deploy. Stack BeluPerfLogsCLientSecond is up to date
```

Reply:
I agree with this, especially "echo" part. But output of "aws lambda update-function-code" seems not needed

Reply:
Agreed that the output of `aws lambda update-function-code` seems very verbose and unnecessary. I'd also change the message from `Deploying Lambda changes to $LAMBDA_ARN` to `Deployed Lambda changes to $LAMBDA_ARN` after the operation was successful. IIRC it's a quick operation and we don't need to inform the user that we're about to perform it